README.md

```diff
@@ -834,6 +835,8 @@ Table of Contents
 
 ## Forms
 
+  * [FabForm].(https://fabform.io/) - Form backend platform for smart developers. Free plan allows 250 form submissions per month. Friendly modern GUI. Integrates with Google Sheets, Airtable, Slack, Email and others.
```

Comment:
The format for links is `[description](url)` yours have a dot between. 

Also please remove the blank line. 

You click top of the PR and view the file to see how it renders.